causalpy/pymc_models.py

```diff
@@ -366,24 +366,52 @@ def build_model(self, X, Z, y, t, coords, priors):
                 shape=(X.shape[0], 2),
             )
 
-    def fit(self, X, Z, y, t, coords, priors):
-        """Draw samples from posterior, prior predictive, and posterior predictive
-        distributions.
+    def sample_predictive_distribution(self, ppc_sampler="jax"):
```

Comment:
Checking you still want the default to be `jax` given that we now don't have this as a project dependency?

Reply:
I think so. It's not going to be invoked in the fit method. But in the notebook i want to suggest jax is the strongly recommended route to sampling the ppc on the instrumental var class.